doc/examples/edges/plot_self_overlapping_polygon.py

```diff
@@ -0,0 +1,60 @@
+"""
+==================================
```

Comment:
Thanks! It would be good to trigger CI again, so I can read the latest version of the gallery example (I'll try and make a few suggestions while reviewing, so you can commit them, which will trigger CI).